yarn.lock

```diff
@@ -0,0 +1,5756 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I'm not sure about the status of the research and experiment for using `yarn`. Wonder if it's the time to commit `yarn.lock`?